diagnostic.md

```diff
@@ -149,7 +162,7 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
+<!--divTwo, divThree,divFour, and divFive will float to the left all on top of each other only divOne will float to the right.-->
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).